doc/appendix-de-kassensichv/examples/vouchers.md

```diff
@@ -1,10 +1,19 @@
-## Vouchers - Implicit Flow
+# Vouchers - implicit flow examples
 
 Examples of single-purpose and multi-purpose requests and responses.
 
-### Single-purpose voucher issuance
+## Table of content
 
-Request (single-purpose voucher issuance **within charge items**):
+- [Single-purpose voucher issuance](#single-purpose-voucher-issuance)
```

Comment:
I would replace the word "issuance" with Issuing of a single-purpose voucher), as issuance is used mostly on a very official, bureaucratic context